readme.md

```diff
@@ -74,6 +74,12 @@ Type: `Function`
 
 A function that accepts an object containing `url`, `index`, and `urls` properties and is expected to return the new filename.
 
+##### downloadInterval
+
+Type: `number`
+
+The delay time between each file download in milliseconds.
```

Comment:
This is the wrong location. Please look at the readme and you'll see that the below code example is for the `rename` option.